File: src/java/1002.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner in = new Scanner(System.in); 
        double r = in.nextDouble();
        double area = 3.14159 * (r * r);

        System.out.printf("A=%.4f\n", area);
    }
}

File: kotlin/1002.kt
import java.util.*

fun main(args: Array<String>) {
    val sc = Scanner(System.`in`);
    
    val r: Double = sc.next().toDouble();
    val area: Double = 3.14159 * (r * r);
    
    println("A=${area.format(4)}");
}

fun Double.format(digits: Int) = "%.${digits}f".format(this)